### ISSUE TITLE
[Feature]: Possibility to unsticky top task tiles

### ISSUE THREAD

**Issue by Crotha** [loathers/TourGuide#203]
### What's the request?

That imminent tasks that require your attention _now_ jump to the top of "Tasks" is awesome. 
But them being stuck up there until they are gone is less so. Especially in unrestricted, a lot can come together that I sometimes just don't want to deal with now (or ever).     
Having the reminders on top is nice, but them scrolling down with you isn't. I would love to have an option or a toggle to just unsticky them even with the danger of overlooking them because I was scrolled down.       

As an example: 
I have currently minimized four (Model Train set - won't change it, DMT Dupe - won't do it inrun, Ghost busting - no need, bad ghost, SIT Course enrollment - won't change), and three big ones I want to be reminded (Kramco, Daylight shaving, autumn-aton) and Boombox. 
Those take up more than half (!) of the full height of the sidebar already and it can become worse unfortunately. It makes scrolling down and looking at other tasks/Resources quite hard.     
 ![image](https://github.com/loathers/TourGuide/assets/47434645/5b6fba9f-d8c0-4a42-ae77-baaada803330)

### Account State

Unrestricted (In-Run)

### Your current TourGuide version

313a0d436cad58350f5d3a3e72600818f7a2c1ae

**Comment by docrostov** [comment]
yeah, this is a persistent problem i do not have a great way to deal with -- i agree that the supernag situation is quite problematic. even in standard, i've absolutely had to zoom out or manage my window explicitly to limit the real estate taken up by the supernags. what you describe here makes sense, but i think that sort of management of priority chaining in the minimization struction might not be possible with how it's currently designed, so it will take a bit of poking. wanted to comment to note that this is on my large scale "things we should fix" list but it also is not going to get dealt with imminently. still, thank you for reporting it, it's good to get validation that it isn't just me having these issues lol
